include/deal.II/base/polynomials_bdm.h

```diff
@@ -54,7 +54,7 @@ DEAL_II_NAMESPACE_OPEN
  *   \begin{pmatrix} f_{y}(x,y) \\ -f_{x}(x,y) \end{pmatrix}$.
  *
  *   The basis used to construct the $BDM_{1}$ shape functions is
- *   @f{align*}
+ *   @f{align*}{
```

Comment:
??? Here and elsewhere, doxygen doesn't require the curly braces. Does it even accept them?